storage/innobase/handler/handler0alter.cc

```diff
@@ -1544,6 +1544,15 @@ innobase_col_to_mysql(
 		field->reset();
 
 		if (field->type() == MYSQL_TYPE_VARCHAR) {
+			if (field->column_format() ==
+				COLUMN_FORMAT_TYPE_COMPRESSED) {
+				/* Skip compressed varchar column when
```

Comment:
It looks like an artifact from the original "compressed columns" patch by Weixiang Zhai.
This code path is never reached.
Fragment removed.